### PR TITLE
Update license headers throughout the repo.

### PR DESCRIPTION
**Pull request by josephperrott**
See individual commits.

While this changes a large number of files in the repo, it does not cause many conflicts/churn in pending work.

```
$ yarn -s ng-dev pr discover-new-conflicts 37205
Requesting pending PRs from Github
Retrieved 605 total pending PRs
Checking 128 PRs for conflicts after a merge of #37205
[========================================] ETA: 0s | 128/128

Result:
2 PR(s) which conflict(s) after #37205 merges:
  - 36425: build: remove source-map-test gulp script
  - 37190: deprecate Bazel builder in @angular/bazel
```

### CODE REVIEW

File: browser-providers.conf.js [hunk 1]
```diff
@@ -177,11 +172,14 @@ var browserstackAliases = {
     return customLaunchers[item].base == 'BrowserStack';
   }),
   'DESKTOP': [
-    'BS_CHROME', 'BS_FIREFOX', 'BS_IE9', 'BS_IE10', 'BS_IE11', 'BS_EDGE',
-  ],
-  'MOBILE': [
-    'BS_ANDROID7', 'BS_WINDOWSPHONE'
+    'BS_CHROME',
```
Comment (IgorMinar) [inline browser-providers.conf.js:175]: it seems that unrelated formatting changes have been included. I think it's fine. it's just unexpected.

**Review by IgorMinar — APPROVED** [review]
lgtm, but let's hold off with this change until after rc.0 so that we don't disrupt the rc-related work. It should be relatively easy to recreate/update this PR, so I hope that delaying the merge a bit is not a big issue.

Reviewed-for: global-approvers

File: browser-providers.conf.js (hunk 1 above)
Comment (josephperrott) [reply browser-providers.conf.js:175]: Yes, it looks like when this file was touched, it was not correctly formatted, so the styling failed.

**Comment by josephperrott** [comment]
@IgorMinar As noted in the PR description, this PR only causes conflicts in 2 PRs (one of them mine, and its a conflict because a file gets removed that modified here).  Since it does not disrupt any work in any other pending PRs I believe it is safe to land immediately.

**Review by IgorMinar — APPROVED** [review]
Reviewed-for: global-approvers
